<!-- -->
<launch>
  <arg name='host' default='localhost'/>
  <arg name='port' default='2000'/>
  <arg name='timeout' default='2'/>
  <arg name="role_name" default="ego_vehicle"/>
  <arg name='resolution_x' default='800'/>
  <arg name='resolution_y' default='600'/>
  <arg name='fov' default='50'/>

  <param name="/carla/host" value="$(arg host)" />
  <param name="/carla/port" value="$(arg port)" />
  <param name="/carla/timeout" value="$(arg timeout)" />

  <node pkg="carla_spectator_camera" type="carla_spectator_camera.py" name="carla_spectator_camera" output="screen">
      <param name="role_name" value="$(arg role_name)"/>
      <param name="resolution_x" value="$(arg resolution_x)"/>
      <param name="resolution_y" value="$(arg resolution_y)"/>
      <param name="fov" value="$(arg fov)"/>
  </node>
</launch>

